examples/resources/nuodbaas_backuppolicy/resource.tf

```diff
@@ -0,0 +1,32 @@
+# A backup policy with minimal configuration
```

Comment:
Nit: You could mention that this policy matches any database in organization named `org` and doesn't delete backups automatically.